qadence/register.py

```diff
@@ -272,14 +356,6 @@ def __eq__(self, other: object) -> bool:
 
 
 def line_graph(n_qubits: int) -> nx.Graph:
-    """Create graph representing linear lattice.
-
-    Args:
-        n_qubits (int): number of nodes in the graph
-
-    Returns:
-        graph instance
-    """
     graph = nx.Graph()
     for i in range(n_qubits):
         graph.add_node(i, pos=(i, 0.0))
```

Comment:
should the docstring be cut off from the function?

Reply:
The reason I removed it was just so it didn't show up in the API reference. This is an auxiliary function that actually should be private, I just didn't feel like refactoring the whole code.